unit/ActorTest.cpp

```diff
@@ -54,3 +56,25 @@ TEST_CASE("Actor should load be able to load appearance, equipment, "
   REQUIRE(actor.GetChangeForm().spawnPoint.rot == NiPoint3{ 1, 2, 4 });
   REQUIRE(actor.GetChangeForm().spawnDelay == 8.0f);
 }
+
+TEST_CASE("Actor's value can be modified", "[Actor]")
```

Comment:
From the docs https://www.creationkit.com/index.php?title=RestoreActorValue_-_Actor
**Negative numbers will be converted to positive so -100 or 100 will have the same effect.**

Thanks for implementing this behavior carefully! But I think it will be better to unit test that. This is a way we guarantee further refactoring will not break this behavior.